t/43-cli-api.t

```diff
@@ -60,6 +60,7 @@ my @host = ('--host', $host);
 # Default options for authentication tests
 my @auth = ('--apikey', 'ARTHURKEY01', '--apisecret', 'EXCALIBUR', @host);
 
+$ENV{MOJO_CONNECT_TIMEOUT} = 1;
```

Comment:
does this mean waiting 1s in tests additionally?

Reply:
Actually we don't need it. I think because "Connection refused" will return immediately in this case. So I removed it again.